community/minutes/2025-03-19.md

```diff
@@ -0,0 +1,79 @@
+---
+tags: [meeting-notes]
+title: '2025-03-19'
+---
+# conda-forge core meeting 2025-03-19
+
+Add new agenda items under the `Your __new__() agenda items` heading
+
+- [Zoom link](https://zoom.us/j/9138593505?pwd=SWh3dE1IK05LV01Qa0FJZ1ZpMzJLZz09)
+- [What time is the meeting in my time zone](https://dateful.com/convert/utc?t=5pm)
+- [Previous meetings](https://conda-forge.org/community/minutes/)
+
+## Attendees
+
+| Name                    | Initials | GitHub ID        | Affiliation                 |
+| ----------------------- | -------- | ---------------  | --------------------------- |
+| Daniel Ching            | DJC      | @carterbox       | cf / NVIDIA                 |
+| Jaime Rodríguez-Guerra  | JRG      | @jaimergp        | Quansight                   |
+| Marius van Niekerk      | MvN      | @mariusvniekerk  | cf / Voltron Data           |
+| Uwe Korn                | UK       | @xhochy          | cf / QuantCo                |
+| Wolf Vollprecht         | WV       | @wolfv           |                             |
+| Isuru Fernando          | IF       | @isuruf          |                             |
+|                         |          |                  |                             |
+|                         |          |                  |                             |
+|                         |          |                  |                             |
+
+X people total
+
+### Standing items
+
+- [ ]
+
+### From previous meeting(s)
+
+- [ ]
+
+### Active votes
+
+- [X] IF: Vote for adding Daniel Nachun to staged-recipes ends in ~6 days
+    - Only 13 votes yet. Need one more vote (quorum needs 27 * 0.5)
```

Comment:
I don't think I got the email, we are using another voting platform, right? Maybe it went to spam :-/

We need to take that into account here.